hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ListCodecTemplate.java

```diff
@@ -190,6 +230,9 @@
     Object removeWithIndex(String name, int index);
 
     /**
+     * Returns the index of the last occurrence of the specified element in this list, or -1 if this list does not
+     * contain the element. More formally, returns the highest index i such that
+     * (value==null ? get(i)==null : value.equals(get(i))), or -1 if there is no such index.
```

Comment:
Can we explain this without get(i) usage, more in terms of the protocol operations?